org.bino3d.bino.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.bino3d.bino
+runtime: org.kde.Platform
+runtime-version: '6.3'
+sdk: org.kde.Sdk
+command: bino
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Play videos from the net
+  - --share=network
+  # OpenGL rendering
+  - --device=dri 
+  # Webcam capturing
```

Comment:
just for completeness: A camera portal exists: https://github.com/flatpak/xdg-desktop-portal/issues/414 it could be worth raising this with upstream so that a more fine-grained access can be provided rather than the wholesale access to all devices.

Reply:
Speaking as upstream, I see two problems with this approach for Bino:

1. The portal mechanisms involve dialogs to get user feedback if I understood correctly, which would work in GUI mode, but not in Virtual Reality mode where you either wear a head mounted display or stand in a large scale VR system.
2. Bino uses Qt Multimedia for camera access, so there is no place in the code to integrate calls to portal APIs -- and to be completely honest, I rewrote Bino based on Qt precisely so that I don't have to deal with platform-specific code such as this...